### ISSUE TITLE
Stuck while downloading Youtube Playlist

### ISSUE THREAD

**Issue by theguyoverthere** [ytdl-org/youtube-dl#7307]
Hello, 

I've tried looking around, but I could not figure out what is causing this. I'm trying to download the Playlist "IntelliJ IDEA Video Tutorials" from https://www.youtube.com/playlist?list=PLPZy-hmwOdEXdOtXdFzyx_XCnrF_oD2Ft using youtube-dl version 2015.10.24, but the download just appears stuck without any error message or progress. I cannot even stop the process using Ctrl+C.

Microsoft Windows [Version 10.0.10240](c) 2015 Microsoft Corporation. All rights reserved.

C:IntelliJ Videos>youtube-dl.exe -citv "https://www.youtube.com/playlist?list=PLPZy-hmwOdEXdOtXdFzyx_XCnrF_oD2Ft" > E:\you.txt
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'-citv', u'https://www.youtube.com/playlist?list=PLPZy-hmwOdEXdOtXdFzyx_XCnrF_oD2Ft']
[debug] Encodings: locale cp1252, fs mbcs, out None, pref cp1252
[debug] youtube-dl version 2015.10.24
[debug] Python version 2.7.10 - Windows-8-6.2.9200
[debug] exe versions: none
[debug] Proxy map: {}

Any idea?
Thanks

**Comment by dstftw** [comment]
What if you also pass `-4`?

**Comment by theguyoverthere** [comment]
Moved forward but unfortunately got stuck again. The new log is pasted below-

C:\IntelliJ Videos>youtube-dl.exe -citv4 "https://www.youtube.com/playlist?list=PLPZy-hmwOdEXdOtXdFzyx_XCnrF_oD2Ft"
[debug] System config: []
[debug] User config: []
[debug] Command-line args: [u'-citv4', u'https://www.youtube.com/playlist?list=PLPZy-hmwOdEXdOtXdFzyx_XCnrF_oD2Ft']
[debug] Encodings: locale cp1252, fs mbcs, out cp437, pref cp1252
[debug] youtube-dl version 2015.10.24
[debug] Python version 2.7.10 - Windows-8-6.2.9200
[debug] exe versions: none
[debug] Proxy map: {}
[youtube:playlist] PLPZy-hmwOdEXdOtXdFzyx_XCnrF_oD2Ft: Downloading webpage
[download] Downloading playlist: IntelliJ IDEA Video Tutorials
[youtube:playlist] playlist IntelliJ IDEA Video Tutorials: Downloading 56 videos
[download] Downloading video 1 of 56
[youtube] L_jXj0XTwSg: Downloading webpage
[youtube] L_jXj0XTwSg: Downloading video info webpage
[youtube] L_jXj0XTwSg: Extracting video information
[youtube] L_jXj0XTwSg: Downloading DASH manifest
[youtube] L_jXj0XTwSg: Downloading DASH manifest
[debug] Invoking downloader on u'https://r19---sn-cvh7zne7.googlevideo.com/videoplayback?upn=JqeNjsMqEmE&itag=22&fexp=9407059%2C9408710%2C9414764%2C9416126%2C9417707&signature=4EDDCDDAF870C3A8A11973855112CE309FBC0986.1F7F8E94F8CA409881A742E60CC7E972E9E60540&sparams=dur%2Cid%2Cinitcwndbps%2Cip%2Cipbits%2Citag%2Clmt%2Cmime%2Cmm%2Cmn%2Cms%2Cmv%2Cpl%2Cratebypass%2Crequiressl%2Csource%2Cupn%2Cexpire&sver=3&mn=sn-cvh7zne7&initcwndbps=67500&requiressl=yes&key=yt6&mime=video%2Fmp4&source=youtube&lmt=1419335284956271&dur=403.678&ratebypass=yes&ip=14.99.48.184&mv=m&pl=20&ipbits=0&ms=au&expire=1445988162&mm=31&id=o-AH5rhpkLsb_6pCYU0DxbDysLNzuJ6gLVM70qXuTNdh58&mt=1445966440'
[download] Destination: Running IntelliJ IDEA for the First Time-L_jXj0XTwSg.mp4
[download]  58.4% of 31.39MiB at 130.52KiB/s ETA 01:42

**Comment by dstftw** [comment]
What do you mean by stuck? How much time passes before you kill the process? What is you connection? Ping something in parallel when you download.

**Comment by theguyoverthere** [comment]
Well by being stuck, I meant that the download did not progress any further, neither was I able to stop the download using Ctrl + C. I'd waited almost for couple of minutes before posting the output above. The download indicator remained at 58.4% and did not complete. I have a stable internet connection - did not lose any packets when I pinged 8.8.8.8 during the download process.

**Comment by dstftw** [comment]
Do you have some AV/personal firewall installed? If you have try adding youtube-dl to the whitelist or disabling them.

**Comment by theguyoverthere** [comment]
Yes I have and I've already tried both options that you have suggested, but nothing seems to work. I use "Kaspersky Internet Security 2015"

**Comment by alltimemarr** [comment]
Mine is stuck too, I'm downloading a Brightcove video:

"frame=16409 fps= 18 q=-1.0 size=  158305kB time=00:10:56.40 bitrate=1975.7kbits/" 
Been like that for some minutes.. I managed to download it two times correctly, other videos form Brightcove. It happened before, but I just restarted it again (wanting to resume, but it just started from scratch again..)

Now it happened twice already with the same video. But I'll try again. Hopefully will work.
